scripts/release/index.ts

```diff
@@ -37,6 +37,10 @@ async function updatePackageJsons(version: string) {
 
     packageJson.version = version;
 
+    if (packageJson.peerDependencies?.["@dso-toolkit/core"]) {
+      packageJson.peerDependencies["@dso-toolkit/core"] = `${version}`;
```

Comment:
De template literal is niet nodig, ik vermoed een overblijfsel toen er nog een `^` stond.

Reply:
Gedaan